libuuid/meson.build

```diff
@@ -31,7 +31,7 @@ endif
 
 thread_dep = dependency('threads')
 
-lib_uuid = both_libraries(
+lib_uuid = library(
```

Comment:
... but this is the reverse change and does indeed support the use case you want, of allowing the distributor to choose whether they want static, shared, or both.

Reply:
Correct. When passing --disable-shared in autotools, the shared library goes away.